lib/ansiblelint/rules/__init__.py

```diff
@@ -174,6 +178,17 @@ def matchyaml(self, file: str, text: str) -> List[MatchError]:
         return matches
 
 
+class RuntimeErrorRule(AnsibleLintRule):
+    """Used to identify errors."""
+
+    id = '902'
+    shortdesc = 'Unexpected internal error.'
+    description = 'This error can be caused by outdated custom rules.'
+    severity = 'VERY_HIGH'
+    tags = ['core']
+    version_added = 'v4.4.0'
```

Comment:
Isn't this a breaking change and should end up in a new major release?

Reply:
Done.